modules/io/src/common/xviz-message-type.js

```diff
@@ -12,12 +12,13 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 export const XVIZMessageType = Object.freeze({
```

Comment:
All caps?

Reply:
yeah, i'll do a pass to make sure other cases do this too.